artifacts/camara-cloudevents/event-subscription-template.yaml

```diff
@@ -0,0 +1,1513 @@
+openapi: 3.0.3
+info:
+  title: Notification Subscription Template
+  description: |
+    This file is a template for CAMARA API explicit subscription endpoint and for Notification model. Additional information are provided in API Design Guidelines document.
+
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: project-email@sample.com
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.1.0-wip
+externalDocs:
+  description: Product documentation at CAMARA
+  url: hhttps://github.com/camaraproject/
+servers:
+  - url: '{apiRoot}/event-type-notification-subscriptions/v0'
+    variables:
+      apiRoot:
+        default: http://localhost:9091
+        description: API root
+tags:
+  - name: event-type notification subscription
+    description: Operation to manage event subscription on event-type event 
+
+paths:
+  /subscriptions:
+    post:
+      tags:
+        - event-type notification subscription
+      summary: 'Create a event-type event subscription'
+      description: Create a event-type event subscription
+      operationId: createEvent-TypeSubscription
+      parameters:
+        - $ref: '#/components/parameters/x-correlator'
+      security:
+        - openId:
+          - api-name:subscriptions:create
+          - api-name:event-type:grant-level
```

Comment:
@bigludo7 Shouldn't this pattern be event-type:subscriptions:create?

Reply:
So here I need to have probably 2 scopes:
 `event-type-notification-subscriptions:create` (This is api_name:action) 
 `event-type-notification-subscriptions:event-type:read` `(this is api-name:event-type:action)

Reply:
@bigludo7 :  If we have the generic type (api_name) in addition to the event type, then what happens when the enduser denies consent for the event-type included scope but agrees to general subscription? I guess for create, we can simply use the event type included scope. That should suffice. WDYT?